lumigator/python/mzai/sdk/tests/int_test_datasets.py

```diff
@@ -27,51 +27,49 @@ def test_get_datasets_remote_ok(lumi_client):
 
 
 def test_dataset_lifecycle_remote_ok(lumi_client, dialog_data):
-    with Path.open(dialog_data) as file:
-        datasets = lumi_client.datasets.get_datasets()
-        before = datasets.total
-        dataset = lumi_client.datasets.create_dataset(dataset=file, format=DatasetFormat.JOB)
-        datasets = lumi_client.datasets.get_datasets()
-        after = datasets.total
-        dataset = lumi_client.datasets.get_dataset(datasets.items[0].id)
-        assert dataset is not None
-        lumi_client.datasets.delete_dataset(datasets.items[0].id)
-        datasets = lumi_client.datasets.get_datasets()
-        assert after - before == 1
+    datasets = lumi_client.datasets.get_datasets()
```

Comment:
would suggest adding a comment here detailing what this test does

Reply:
Ok